manifests/latest/vsphere-7.0u1/vanilla/deploy/generate-signed-webhook-certs.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# File originally from https://github.com/istio/istio/blob/release-0.7/install/kubernetes/webhook-create-signed-cert.sh
+set -e
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+    cat <<EOF
+Usage: Generate certificate suitable for use with an webhook service.
+
+This script uses k8s' CertificateSigningRequest API to a generate a
+certificate signed by k8s CA suitable for use with sidecar-injector webhook
+services. This requires permissions to create and approve CSR. See
+https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster for
+detailed explanation and additional instructions.
+
+The server key/cert k8s CA cert are stored in a k8s secret.
+EOF
+    exit 1
+fi
+
+service=scvalidation-webhook-svc
+secret=scvalidation-webhook-certs
+namespace=kube-system
```

Comment:
what if csi driver is not installed in `kube-system` namespace?

Reply:
all of our YAML files are currently uses `kube-system` we need to create service in the namespace where controller pod is running.

if we are deploying driver in the different namespace we need to change this. I can supply this as an argument and default to `kube-system` if not specified anything to script.